__tests__/__unit__/shared/utils.unit.test.ts

```diff
@@ -0,0 +1,65 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import { ZoweUSSNode } from "../../../src/uss/ZoweUSSNode";
+import * as vscode from "vscode";
+import { Session, IProfileLoaded, Logger } from "@zowe/imperative";
+import * as sharedUtils from "../../../src/shared/utils";
+import { Profiles } from "../../../src/Profiles";
+
+const session = new Session({
+    user: "fake",
+    password: "fake",
+    hostname: "fake",
+    protocol: "https",
+    type: "basic",
+});
+const profileOne: IProfileLoaded = {
+    name: "aProfile",
+    profile: {},
+    type: "zosmf",
+    message: "",
+    failNotFound: false
+};
+
+const mockLoadNamedProfile = jest.fn();
+
+Profiles.createInstance(Logger.getAppLogger());
+
+describe("Shared Utils Unit Tests - Function node.labelHack()", () => {
```

Comment:
labelHack? can we use a different name than hack? it feels like a temporary solution.

Reply:
Maybe an issue should be opened for this? Because sometime we need to figure out why changing the label of a node is required to get VSCode to update the node.

This function is doing the renaming to cause the update, by adding a space. What would you call it?

Reply:
node.labelRenamed? What do you think?

Reply:
Maybe something that says it isn't really renaming the node...labelForceUpdate?

Is it okay?

Reply:
labelUpdate? Now adding force makes it feel like its a bug. hehe. naming a variable is hard 😄

Reply:
Wellll it kinda is a bug...idk if its ours or VSCode's XD

Maybe labelRefresh? Then it doesn't sound like we are actually changing the label significantly...

Ya I'm not great at this XD

Reply:
YEah. labelRefresh sounds good. go for that 👍